packages/Python/lldbsuite/test/lang/swift/variables/swift/TestSwiftTypes.py

```diff
@@ -162,7 +163,7 @@ def do_test(self):
                 'float64',
                 'value = 2.5'])
         float80_unsupported_platforms = ["ios"]
```

Comment:
Can you add your unsupported platform here to the array? (maybe the check can be made a little more specific to just limit Linux [I doubt there are other ppc64 platform supporting swift out there, but still]).

Reply:
Initially, I tried having "ppc64le" in the unsupported platforms list like below, but that didn't work:-
	float80_unsupported_platforms = ["ios","ppc64le"]
This is because the platform string returned by self.getPlatform() in our case is "linux", and not "ppc64le".

Did you mean to include something like this?
        float80_unsupported_platforms = ["ios","linux"]

This might effectively block all linux platforms including x86 platforms.
Hence I had put the check separately only for ppc64le in the subsequent 'if' condition.

Please suggest.

Reply:
Hi @dcci anything on this?

Reply:
You might want to refactor that test to allow both an architecture and a OS to be specified.

Reply:
Done, please review the latest changes and approve this PR.